.azure-pipelines/env.yml

```diff
@@ -142,6 +142,59 @@ jobs:
     displayName: Set development/release env
     name: state
 
+  - bash: |
+      # TODO(phlax): move this to a script to ensure proper linting etc
+      set -e
+
+      # Run everything in postsubmit
+      if [[ "$(Build.Reason)" != "PullRequest" ]]; then
+          echo "##vso[task.setvariable variable=build;isoutput=true]true"
+          echo "##vso[task.setvariable variable=checks;isoutput=true]true"
+          echo "##vso[task.setvariable variable=docker;isoutput=true]true"
+          echo "##vso[task.setvariable variable=packaging;isoutput=true]true"
+          exit 0
+      fi
+
+      RUN_BUILD=true
+      RUN_CHECKS=true
+      RUN_DOCKER=true
+      RUN_PACKAGING=true
+      if [[ $(changed.mobileOnly) == true || $(changed.docsOnly) == true ]]; then
+          RUN_BUILD=false
+          RUN_DOCKER=false
+      fi
+      if [[ "$(changed.mobileOnly)" == true || $(changed.docsOnly) == true || $(changed.examplesOnly) == true ]]; then
```

Comment:
nit: unnecessary quotation marks or others lost their quotation marks?

Reply:
Yeah, should be consistent.

Reply:
ill add, boolean logic is a real nitghtmare with azp - a lot of things end up _having_ to be strings

Reply:
updated so they are consistent